Remotion/ObjectBinding/Web.Development.WebTesting.TestSite.Shared/Controls/BocAutoCompleteReferenceValueWebService.asmx.cs

```diff
@@ -86,7 +96,11 @@ public IconProxy GetIcon (string businessObjectClass, string businessObject, str
             .Take(count)
             .Select(e => new BusinessObjectWithIdentityProxy { DisplayName = $"testlist Person {e}", UniqueIdentifier = e.ToString() })
             .ToArray();
-        return BocAutoCompleteReferenceValueSearchResult.CreateForValueList(resultItems, completionSetOffset + count < 20);
+
+        return BocAutoCompleteReferenceValueSearchResult.CreateForValueList(
+            resultItems,
+            completionSetOffset + count < 20,
+            (completionSetOffset + completionSetCount).ToString());
```

Comment:
Inconsitent with the one below. Should this be `count` instead of `completionSetCount`